### DISCUSSION TITLE
(read before posting) Welcome to Et Futurum Discussions! Share your feature requests and improvement ideas here!

### DISCUSSION THREAD

**Discussion by Roadhog360** [Roadhog360/Et-Futurum-Requiem#46] [Ideas]
# 👋 Welcome!

## Note that content already backported that is missing features or functionality is a BUG and should be reported in the issues tab. 
  
  Use the "Ideas" category to request a new feature to port. Block and item requests are boring, request other features like structures or game mechanics instead!
  
  If you want to see an existing idea be improved, use the "Improvements" category!
  
  Like someone's idea? Click the little upvote icon on top next to the title!
  
  Make sure to isolate your idea, it shouldn't be too broad or all-encompassing, there should be a reasonable goal. EG Ask for a specific mob or set of related content. Don't ask for stuff like "all the mobs".
  Good example: Add all of the illagers + woodland mansion + pillager outpost
  Bad example: Add all modern structures

No more "meow" descriptions. Actually describe what you're asking for, give a brief description of the feature.

**Comment by GregoriusT** [comment]
Since this is technically Et-Futurum-Requiem, should the project not be named exactly that, in order to prevent confusion when someone google searches for the GitHub Page and finds the old Et-Futurum?
